product_harmonized_system/models/product_category.py

```diff
@@ -14,6 +14,7 @@ class ProductCategory(models.Model):
         "hs.code",
         string="H.S. Code",
         ondelete="restrict",
+        company_dependent=True,
```

Comment:
for a specific product the common code is the same for ALL countries around the world, the last digits can vary per country.
In any case making this company dependent doesn't make any sense.

Imagine you are in a mullti-company setup with e.g. 10 legal entities in spain and shared products and entities in other countries.

Adding a product with his common code will apply to  all of these entities.
The 'local_code' part must be shared between all 10 spanish entities.

This PR should be reworked by changing the local_code part and via via company_dependent but via (to be programmed) 'country_dependent' logic.